lib/manageiq/providers/ovirt/legacy/event_monitor.rb

```diff
@@ -0,0 +1,50 @@
+require 'ovirt'
```

Comment:
Any reason you chose to move them to `lib/`?
I think this all could easily go to `app/models/manageiq/providers/redhat`